2023/assignments/bibek/bibek.md

```diff
@@ -0,0 +1,144 @@
+# <p align='center'>Git Workshop 2023 <p>
+
+### <p align='center'>`<your_name>`</p>
+
+<p align="center">
+    <a href="#" target="blank"
```

Comment:
@th3-roGue-deV1l, if you have social media links, kindly replace them. Otherwise, remove any existing links.